### PR TITLE
Working on Vive shutdown issues

### PR DESCRIPTION
**Pull request by jherico**
Fixing some problems while trying to resolve [FB18069](https://highfidelity.manuscript.com/f/cases/18069)

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13258/HighFidelity-Beta-PR14073-72fc686.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13258/HighFidelity-Beta-Interface-PR14073-72fc686.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13258/HighFidelity-Beta-PR14073-72fc686.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13258/HighFidelity-Beta-Interface-PR14073-72fc686.dmg

**Comment by qlimguicheteau** [comment]
Now Testing

**Comment by qlimguicheteau** [comment]
Fail @conklin94122 

Computer 1: No crash screens occurred on my end of the test.

Computer 2: Lower spec machine, and had a few crashes, but the ratio was improved to about 2 in 10, so if this is related to what was changed, its a fail, if its a different reason for crashing, its a pass.

[Vive2 crash Log hifi-log_2018-09-25_08.47.07.txt](https://github.com/highfidelity/hifi/files/2415718/Vive2.crash.Log.hifi-log_2018-09-25_08.47.07.txt)

Its last line before the crash on restart was.
[09/25 08:46:55] [WARNING] [default] DependencyManager::destroy(): class AvatarManager was not properly destroyed!

And there was a Backtrace mention:
[09/25 08:46:53] [CRITICAL] [hifi.scriptengine] [Backtrace]
[09/25 08:46:53] [CRITICAL] [hifi.scriptengine]     off() at file:///C:/Program Files/High Fidelity - PR14073/scripts/system/commerce/wallet.js:519
[09/25 08:46:53] [CRITICAL] [hifi.scriptengine]     shutdown() at file:///C:/Program Files/High Fidelity - PR14073/scripts/system/commerce/wallet.js:534
[09/25 08:46:53] [CRITICAL] [hifi.scriptengine]     <global>() at -1

File: libraries/audio-client/src/AudioClient.cpp [hunk 1]
```diff
@@ -278,8 +285,8 @@ void AudioClient::customDeleter() {
     _shouldRestartInputSetup = false;
 #endif
     stop();
-    _checkDevicesTimer->stop();
-    _checkPeakValuesTimer->stop();
+    //_checkDevicesTimer->stop();
```
Comment (samcake) [inline libraries/audio-client/src/AudioClient.cpp]: comment can go ?

**Comment by jherico** [comment]
@conklin94122 @qlimguicheteau 

The two crashes appear to be completely different from one another.  One is a crash in the AMD display driver that has no usable information in the stack trace.  The other one is a crash that I was seeing very intermittently in my testing.  I've found another possible issue leading to the second crash, so I'm updating the PR shortly.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13323/HighFidelity-Beta-PR14073-15de059.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13323/HighFidelity-Beta-PR14073-15de059.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13323/HighFidelity-Beta-Interface-PR14073-15de059.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13323/HighFidelity-Beta-PR14073-15de059.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13323/HighFidelity-Beta-Interface-PR14073-15de059.exe

**Comment by qlimguicheteau** [comment]
Now Testing

**Comment by qlimguicheteau** [comment]
FAIL @conklin94122 

Again a descrepancy crash, I flew outside of an area, then attempted to launch from a good distance away. 
The interface came up for about a second, then immediately shut down.
/-127.005,122.203,1045.06/0,-0.235715,0,0.971822
Location

[OpeningCrash hifi-log_2018-09-25_15.49.14.txt](https://github.com/highfidelity/hifi/files/2417214/OpeningCrash.hifi-log_2018-09-25_15.49.14.txt)

**Pull request merged**